Explorer/Assets/DCL/LOD/Components/SceneLODInfo.cs

```diff
@@ -11,21 +12,19 @@ namespace DCL.LOD.Components
     public struct SceneLODInfo
     {
         public byte CurrentLODLevel;
-        internal LODAsset? CurrentLOD;
-        internal LODAsset? LastSuccessfulLOD;
+        internal LODAsset CurrentLOD;
```

Comment:
Considering you already have the getter methods make these fields private pls

Reply:
Made them public